pyfiglet/__init__.py

```diff
@@ -819,7 +852,11 @@ def main():
         # Set stdout to binary mode
         sys.stdout = sys.stdout.detach()
 
-    sys.stdout.write((r + '\n').encode('UTF-8'))
+    foreground_color = __get_ascii_color(opts.foreground_color, isBackground=False)
+    background_color = __get_ascii_color(opts.background_color, isBackground=True)
+    colors_string = foreground_color + background_color    
+    default_colors = __get_ascii_color('DEFAULT', True) + __get_ascii_color('DEFAULT', False)
+    sys.stdout.write((colors_string + (r + '\n') + default_colors).encode('UTF-8'))
```

Comment:
In my view, it's quite important not to do anything to the terminal state unless the user requests it. So the default behaviour if changing the colour is not specified should be to leave the colour as it is (i.e, not print any escape characters). I suggest achieving this by defaulting the option to `None` and doing `if color is not None: [... use escape sequences ...]` otherwise, leaving the behaviour unchanged.

Reply:
You're right, I don't know why I did it that way....